image.yaml

```diff
@@ -131,15 +131,18 @@ ports:
 modules:
       repositories:
           - git:
-                  url: https://github.com/jboss-openshift/cct_module.git
-                  ref: master
+              url: https://github.com/jboss-openshift/cct_module.git
+              ref: master
+          - git:
+              url: https://github.com/jboss-container-images/jboss-kie-modules.git
+              ref: master
       install:
-          - name: os-kieserver-launch
-          - name: os-kieserver-launch
-          - name: os-kieserver-probes
-          - name: os-kieserver-s2i
-          - name: os-kieserver-webapp
-          - name: os-kieserver-chmod
+          - name: os.kieserver.launch
```

Comment:
Can we use a more descriptive name, e.g. like Java package name for these?  The goal being to be able to easily identify from which repository the module is pulled, e.g. jboss.kie.kieserver.launch: kieserver.launch module from jboss-kie repsoitory.  If you have a better idea for naming, feel free to adjust as necessary.  With the example I provided, you might also want to add a name to the repo, e.g. jboss.kie.repo.  Having a name is a critical piece required to override the module location for development purposes, e.g. overrides.yaml:
```yaml
modules:
  repositories:
  - name: jboss.kie.repo
    path: /home/me/src/git/jboss-container-images/jboss-kie-modules
```

Reply:
@rcernich I like this idea (changing the names), but can we do this as follow-up PRs? There's already a lot in this jira...

Reply:
After speaking with @rcernich on IRC, the renaming of modules (and adding repo names) will be done in subsequent PRs. We need to move these PRs so other work can progress.